### PR TITLE
DM-1415: [DM-1599] Allow metrics to be exported as .xlsx

### PR DESCRIPTION
**Pull request by emyl3**
### JIRA issue link
[DM-1415](https://agile6.atlassian.net/secure/RapidBoard.jspa?rapidView=22&projectKey=DM&modal=detail&selectedIssue=DM-1415&quickFilter=31): [[DM-1599]](https://agile6.atlassian.net/secure/RapidBoard.jspa?rapidView=22&projectKey=DM&modal=detail&selectedIssue=DM-1599&quickFilter=31)

## Description - what does this code do?
This PR adds the ability for an admin to export the data in the "Metrics" tab as a  `.xlsx` file.

## Testing done - how did you test it/steps on how can another person can test it 
- Log in as an admin user
- Click the "Metrics" tab
- Ensure the values are as expected...
- Click "Export as .xlsx"
- A file download prompt should open
- Download the file and make sure the data is equivalent to the one seen in the "Metrics" tab

## Definition of done
- [ ] Unit tests written (if applicable)
- [X] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [X] A link has been provided to the originating JIRA issue
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: app/admin/dashboard.rb [hunk 1]
```diff
@@ -163,35 +287,24 @@
       end # tab
 
       tab :metrics do
+        # export .xlsx button
+        form action: export_metrics_path(), method: :get, style: 'text-align: right' do |f|
```
Comment (ahay-agile6) [inline app/admin/dashboard.rb:291]: for shame! `()` :smile: lol

File: app/admin/dashboard.rb [hunk 2]
```diff
@@ -163,35 +287,24 @@
...
+          f.input :submit, :type => :submit, :value => 'Export as .xlsx', style: 'margin-bottom: 1rem'
```
Comment (ahay-agile6) [inline app/admin/dashboard.rb:292]: oh, and let's get rid of these "rockets", just use `type: :submit`, etc.

File: spec/features/admin_spec.rb [hunk 3]
```diff
@@ -56,19 +56,40 @@
...
     end
   end
 
+  it 'should show allow admin to download metrics as .xlsx file' do
+    login_as(@admin, scope: :user, run_callbacks: false)
+    visit '/admin'
+
+    click_link('Metrics')
+    export_button = find(:css, "input[type='submit']")
+    export_button.click
```
Comment (emyl3) [inline spec/features/admin_spec.rb:87]: ⚠️ I believe this downloads a file locally whenever we run this... Not sure if we want to do that... 🤔

File: app/admin/dashboard.rb (hunk 1 above)
Comment (emyl3) [reply app/admin/dashboard.rb]: oops!! hahaha habits are hard to break -- my bad 😅

File: app/admin/dashboard.rb (hunk 2 above)
Comment (emyl3) [reply app/admin/dashboard.rb]: 🙆‍♂️ haha thank you!

File: spec/features/admin_spec.rb (hunk 3 above)
Comment (ahay-agile6) [reply spec/features/admin_spec.rb:87]: hmmm can we clear the file after the test?
`FileUtils.rm_rf(<file here>)`

Comment (emyl3) [reply spec/features/admin_spec.rb:93]: yay thank you for the guidance :)

**Pull request merged**